### ISSUE TITLE
Generated files are poorly formatted

### ISSUE THREAD

**Issue by NielsKersic** [vegardit/prisma-generator-nestjs-dto#92]
When I generate the classes from my Prisma schema, the outputted files are very weirdly/poorly formatted.
Some files have empty lines before the class starts, others are unnecessarily or inconsistently indented.

The problem is easily fixed by running `npm run format` but it would be nice if this step would not be necessary.

**Comment by benkroeger** [comment]
I agree on the missing formatting step in the generator. However, this was built under the assumption that most codebases have their own formatting rules and processes anyways.

I'm happy to review a PR that introduces `prettier` as an optional peer dependency - and if available - executed at the end of file generation.
